.github/workflows/build-wavefront-celery_worker-develop.yaml

```
@@ -0,0 +1,107 @@
+
+  ACR_REGISTRY_NAME: rootflo
+  ACR_REGISTRY: rootflo.azurecr.io
+  ACR_REPOSITORY: wavefront-celery_worker
+
+jobs:
+  build-push-artifact:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v3"
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)